### PR TITLE
Improve the `PDFObjects` class

### PR DESCRIPTION
**Pull request by Snuffleupagus**
*Please refer to the individual commit messages for additional details.*

### CODE REVIEW

File: src/display/api.js [hunk 1]
```diff
@@ -91,7 +91,7 @@ const DefaultStandardFontDataFactory =
  */
 
 /**
- * @type IPDFStreamFactory
+ * @type {IPDFStreamFactory}
```
Comment (Snuffleupagus) [inline src/display/api.js:94]: This is unrelated to the other changes, but I happened to notice this typo and thought that a separate commit/PR for *only* this changed was excessive :-)

**Review by timvandermeij — APPROVED** [review]

**Comment by timvandermeij** [comment]
Nice improvements; thanks!

**Pull request merged**